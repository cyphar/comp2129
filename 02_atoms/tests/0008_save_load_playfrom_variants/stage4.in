LOAD save1.as
PLAYFROM 3
DISPLAY
STAT
UNDO
DISPLAY
STAT
