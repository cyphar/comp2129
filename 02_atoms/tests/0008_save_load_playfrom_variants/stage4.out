Game Loaded

Game Ready
Blue's Turn


+-----------------------+
|  |  |G1|  |  |  |  |  |
|R1|  |  |P1|  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
+-----------------------+

Player Red:
Grid Count: 1

Player Green:
Grid Count: 1

Player Purple:
Grid Count: 1

Player Blue:
Grid Count: 0

Player Yellow:
Grid Count: 0

Purple's Turn


+-----------------------+
|  |  |G1|  |  |  |  |  |
|R1|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |  |
+-----------------------+

Player Red:
Grid Count: 1

Player Green:
Grid Count: 1

Player Purple:
Grid Count: 0

Player Blue:
Grid Count: 0

Player Yellow:
Grid Count: 0

